waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -606,6 +619,10 @@ proc startOnchainSync(
 
   # static block chunk size
   let blockChunkSize = 2_000
+  # delay between rpc calls to not overload the rate limit
```

Comment:
Do we know what is the rate limit or it depends on every RPC vendor?

Reply:
depends on every rpc provider, this is quite lenient